### ISSUE TITLE
VoiceOver Doesnt read the description of THIS PLAN

### ISSUE THREAD

**Issue by sraarum87** [veshu/accessibility-bughunt-bluesource#25]
**Describe the bug**
VoiceOver Doesnt read the description of THIS PLAN. User will not have additional information about "This Plan" button when clicked on "i"

**To Reproduce**
1. Go to Home page
2. Click on 'This Plan: i"
3. Scroll down or press tab. This doesnt read the description "If you have additional Wellmark coverages and would like to access them select "Other Plans".
4. See error screenshot

**Expected behavior**
Description of "i" should be read out by VoiceOver when down arrow or tab is pressed.

Device: Laptop
Operating System: MacOS 10.14.6
Browser(with version): Chrome Version 78.0.3904.70 (Official Build) (64-bit)

WCAG Category: 3. Understandable
WCAG Level: A
WCAG Success Criterion 3.3.2 Labels or Instructions
**Tools used**: VoiceOver for MacOS

**Attachments**
https://drive.google.com/drive/folders/1kq6i_WJpRbQa0ECXj8EtFO_44mmPjwrM
